package.json

```diff
@@ -61,15 +61,43 @@
           "scope": "window"
         },
         "sonarlint.trace.server": {
-          "type": "string",
-          "enum": [
-            "off",
-            "messages",
-            "verbose"
-          ],
           "default": "off",
           "description": "Traces the communication between VS Code and the SonarLint language server.",
-          "scope": "window"
+          "scope": "window",
+          "anyOf": [
+            {
+              "type": "string",
+              "enum": [
+                "off",
+                "messages",
+                "verbose"
+              ],
+              "default": "off"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "verbosity": {
+                  "type": "string",
+                  "enum": [
+                    "off",
+                    "messages",
+                    "verbose"
+                  ],
+                  "default": "off"
+                },
+                "format": {
+                  "type": "string",
+                  "enum": [
+                    "text",
+                    "json"
```

Comment:
Not part of the ticket, but I found it while looking at other extensions. It should allow to use https://github.com/microsoft/language-server-protocol-inspector.